Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs

```diff
@@ -208,6 +217,32 @@ private InputActionAsset GetAsset()
             return m_State.serializedObject?.targetObject as InputActionAsset;
         }
 
+        private void SetObjectFieldEnabled(bool enabled)
+        {
+            // Update object picker enabled state based off editor play mode
+            if (m_RootVisualElement == null) return;
```

Comment:
Minor : I'd prefer to see this on two lines but thats just a style preference.

Reply:
I realised, fixing with null propagation to avoid this completely.

Reply:
Fixed in https://github.com/Unity-Technologies/InputSystem/pull/1866/commits/9a1406949e81434bcdc93390a271e228abe91f60